letsencrypt-auto-source/pieces/bootstrappers/rpm_python3.sh

```diff
@@ -0,0 +1,25 @@
+# If new packages are installed by BootstrapRpmPython3 below, this version
+# number must be increased.
+BOOTSTRAP_RPM_PYTHON3_VERSION=1
+
+BootstrapRpmPython3() {
+  # Tested with:
+  #   - CentOS 6
+
+  InitializeRPMCommonBase
+
+  # EPEL uses python34
+  if $TOOL list python34 >/dev/null 2>&1; then
+    python_pkgs="python34
+      python34-devel
+      python-virtualenv
```

Comment:
This is largely copied from the existing bootstrap script so you don't have to change it if you want, but I don't think we need `python34-tools` and `python-pip`. If we're using `python3 -m venv`, I don't think we need this package either.